product_docs/docs/eprs/7/08_xdb_cli/03_xdb_cli_commands/36_creating_subscription_cli.mdx

```diff
@@ -46,6 +47,64 @@ The file containing the publication server login information of the publication
 
 Comma-separated list of filter IDs identifying the filter rules from the set of available table filters to enable on the corresponding tables in the new subscription. Use the `printpubfilterslist` command to obtain the filter IDs for the available filter rules in the publication (see [Printing a list of filters in a Publication](17_print_publications_filters_list/#print_publications_filters_list)). Don't use white space between the comma and filter IDs.
 
+`customcolmap`
+
+Enter the column mappings between different but compatible columns in the source and target database. Table and schema qualified column names can be used in mappings, or a regular expression can be used to implement a mapping rule on multiple columns. 
```

Comment:
The description for this parameter starts with a verb to indicate the action the user is to take with this parameter.  I don't have a problem with this style per se, however, it is different than how the descriptions for the previous command options on this page are written.  Ideally, the style should be consistent.